src/main/java/properties/TrelloTestProperties.java

```diff
@@ -0,0 +1,43 @@
+package properties;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class TrelloTestProperties {
+    private static final Properties properties = new Properties();
+    private static final File file = new File("src/test/resources/Config.properties");
+
+    public TrelloTestProperties() {
+        InputStream input = null;
```

Comment:
Instead of constructor use static initialization! Use try with resources. Throw exception inside catch if you cannot read from property file.

Reply:
replaced constructor with static block. Changed try to try-with-resources. Added exception in catch block